lib/galaxy_test/api/vault_conf.yml

```diff
@@ -0,0 +1,13 @@
+type: database
```

Comment:
If you're configuring Galaxy it should move to an integration test

Reply:
Done